common/ManagementTestShared/Redesign/InherentCheckTests.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Azure.ResourceManager.TestFramework
+{
+    public sealed class InherentCheckTests
+    {
+        private string[] ExceptionList { get; set; }
+
+        private const string TestFrameworkAssembly = "Azure.ResourceManager.TestFramework";
+        private const string AssemblyPrefix = "Azure.ResourceManager.";
+        private const string TestAssemblySuffix = ".Tests";
+
+        [Test]
+        public void InherentCheck()
```

Comment:
Should be `InheritanceCheck` or more detailed `ValidateInheritanceForResourceAndCollectionSuffix`

Reply:
Renamed to ValidateInheritanceForResourceAndCollectionSuffix